swift_browser_ui_frontend/src/components/CreateFolderModal.vue

```diff
@@ -1,58 +1,60 @@
-    <c-card-content>
-      <p class="info-text is-size-6">
-        {{ $t("message.container_ops.norename") }}
-      </p>
-      <b-field
-        custom-class="has-text-dark"
-        :label="$t('message.container_ops.containerName')"
-      >
-        <b-input
-          v-model="folderName"
-          name="foldername"
-          aria-required="true"
-          :disabled="!create"
-          data-testid="folder-name"
-        />
-      </b-field>
-      <b-field
-        custom-class="has-text-dark"
-        :label="$t('message.tagName')"
-      >
-        <b-taginput
-          v-model="tags"
-          ellipsis
-          maxlength="20"
-          has-counter
-          rounded
-          type="is-primary"
-          :placeholder="$t('message.tagPlaceholder')"
-          :confirm-keys="taginputConfirmKeys"
-          :on-paste-separators="taginputConfirmKeys"
-          data-testid="folder-tag"
-        />
-      </b-field>
-      <p class="info-text is-size-6">
-        {{ $t("message.container_ops.createdFolder") }}
-        <b>{{ $t("message.container_ops.myResearchProject") }}</b>
-      </p>
-      <c-link
-        :href="`https://my.csc.fi/myProjects/project/${currentProjectID}`"
-        underline
-        target="_blank"
-      >
-        {{ $t("message.container_ops.viewProjectMembers") }}
-        <i class="mdi mdi-open-in-new" />
-      </c-link>
-    </c-card-content>
+  <c-card class="add-folder">
+    <div class="modal-content-wrapper">
+      <h3 class="title is-3 has-text-dark">
+        {{
+          create
+            ? $t("message.container_ops.addContainer")
+            : $t("message.container_ops.editContainer") + folderName
+        }}
+      </h3>
+      <c-card-content>
+        <p class="info-text is-size-6">
+          {{ $t("message.container_ops.norename") }}
+        </p>
+        <b-field
+          custom-class="has-text-dark"
+          :label="$t('message.container_ops.folderName')"
+        >
+          <b-input
+            v-model="folderName"
+            name="foldername"
+            aria-required="true"
+            :disabled="!create"
+            data-testid="folder-name"
+          />
+        </b-field>
+        <b-field
+          custom-class="has-text-dark"
+          :label="$t('message.tagName')"
+        >
+          <b-taginput
+            v-model="tags"
+            ellipsis
+            maxlength="20"
+            has-counter
+            rounded
+            type="is-primary"
+            :placeholder="$t('message.tagPlaceholder')"
+            :confirm-keys="taginputConfirmKeys"
+            :on-paste-separators="taginputConfirmKeys"
+            data-testid="folder-tag"
+          />
+        </b-field>
+        <p class="info-text is-size-6">
+          {{ $t("message.container_ops.createdFolder") }}
+          <b>{{ $t("message.container_ops.myResearchProject") }}</b>
+        </p>
+        <c-link
+          :href="`https://my.csc.fi/myProjects/project/${currentProjectID}`"
```

Comment:
The link doesn't work with internal project ID used by Pouta. It will work with the project name from Vuex store, located in `$store.state.active.name` or `this.$store.state.active.name` in the computed variable.

Project name contains ID used by MyCSC, but it's of a long form `project_XXXXXX`. Removing the `project_` part will make it usable in the link.